manifests/07-downloads-deployment.yaml

```diff
@@ -85,32 +85,36 @@ spec:
           temp_dir = tempfile.mkdtemp()
           print('serving from {}'.format(temp_dir))
           os.chdir(temp_dir)
-          for arch in ['amd64']:
+          for arch in ['amd64', 'arm64', 'ppc64le', 's390x']:
```

Comment:
Have reference for where these come from handy?

Reply:
Perhaps I just need a referesher about the process that spits out the `cli-artifacts`.

Reply:
> Perhaps I just need a referesher about the process that spits out the `cli-artifacts`.

[Dockerfile][1] hits [this Makefile line][2].  But with this approach, there's no reason we couldn't fill in [all the GOOS and GOARCH known by Go][3].

[1]: https://github.com/openshift/oc/blob/386b42b6a3ce0c3060f5c95b9fea5f464745f90a/images/cli-artifacts/Dockerfile.rhel
[2]: https://github.com/openshift/oc/blob/386b42b6a3ce0c3060f5c95b9fea5f464745f90a/Makefile#L99
[3]: https://golang.org/doc/install/source#environment